### PR TITLE
get admin details implemented on endpoint /admin

### PR DESCRIPTION
**Pull request by Gurdeep475**
## Issue that this pull request solves

 Closes: #476 

## Proposed changes
added simple if the check for adminType and fetched the details of admin using email attacked with the request body.

### Brief description of what is fixed or changed
endpoint /admin now returns single logged-in admin details for non-super admins
## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
image

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request
# Consider it under GSSOC`22

### CODE REVIEW

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Gurdeep475 This is an existing api for getting the list of admins. Please make a separate api for getting the details of the loggedin admin. [ You can name it under `getAdmin.js` ]

**Comment by Gurdeep475** [comment]
> @Gurdeep475 This is an existing API for getting the list of admins. Please make a separate api for getting the details of the logged-in admin. [ You can name it under `getAdmin.js` ]

But the original issue asked to implement in the existing /admin API endpoint.

File: backend/app/routes/admin/getAdmins.js [hunk 1]
```diff
@@ -26,6 +26,13 @@ const getAdminsAggregate = (match, page) => {
 module.exports = async (req, res, next) => {
   const page = req.query.page || 1;
   const adminType = req.query.type;
+
+  if(adminType === 'admin'){
+    const adminDetails = await Admin.find({email: req.body.email});
```
[the remark this answers is not in the document]
Comment (Gurdeep475) [reply backend/app/routes/admin/getAdmins.js]: @Kajol-Kumari I have made the necessary changes according to the instructions provided.

**Review by Kajol-Kumari — APPROVED** [review]
LGTM 👍

**Pull request merged**